src/js/view/utils/skin.js

```diff
@@ -274,3 +277,20 @@ export function handleColorOverrides(playerId, skin) {
         }
     }
 }
+
+function isTransparent(color) {
```

Comment:
This method is overkill for a workaround. Just check the default @radium-v uses and 'transparent'.